typology-register/src/redis-client.ts

```diff
@@ -45,7 +45,8 @@ const initializeRedis = async (
   return client;
 };
 
-const cleanStore = async (client: RedisClient): Promise<any> =>
+// TODO: Should we need to return a number? We should return void I think @joey
```

Comment:
Think that the Redis method flushdb returns a number, so we probably meant to just return that same number. Depending on the way we intend to implement error handling, this method will have to follow suite - right now, returning 0 if there's an error is incorrect.